src/main/cliapp/src/components/GenericDataTable/GenericDataTable.js

```diff
@@ -260,37 +261,37 @@ export const GenericDataTable = (props) => {
 					editMode= "row" onRowEditInit= {onRowEditInit} onRowEditCancel= {onRowEditCancel}
 					onRowEditSave= {onRowEditSave} editingRows={editingRows} onRowEditChange={onRowEditChange}
 					sortMode="multiple" removableSort={true} onSort={onSort} multiSortMeta={tableState.multiSortMeta}
-					onColReorder={colReorderHandler} reorderableColumns= {true} 
+					onColReorder={colReorderHandler} reorderableColumns= {true}
 					resizableColumns= {true} columnResizeMode="expand" showGridlines= {true} onColumnResizeEnd={handleColumnResizeEnd}
 					paginator={!isInEditMode} totalRecords={totalRecords} onPage={onLazyLoad} lazy= {true} first={tableState.first}
 					paginatorTemplate="CurrentPageReport FirstPageLink PrevPageLink PageLinks NextPageLink LastPageLink RowsPerPageDropdown"
 					currentPageReportTemplate="Showing {first} to {last} of {totalRecords}"
-					rows={tableState.rows} rowsPerPageOptions={[10, 20, 50, 100, 250, 1000]} 
+					rows={tableState.rows} rowsPerPageOptions={[10, 20, 50, 100, 250, 1000]}
 					rowClassName = {(props) => getRowClass(props)}>
 					{props.isEditable &&
-						<Column field='rowEditor' rowEditor style={{maxWidth: '7rem', minWidth: '7rem'}} filter filterElement={rowEditorFilterNameHeader} showFilterMenu={false}
-							headerStyle={{ width: '7rem', position: 'sticky' }} bodyStyle={{ textAlign: 'center' }} frozen headerClassName='surface-0'
+						<Column field='rowEditor' rowEditor style={{maxWidth: '3rem', minWidth: '3rem', padding: '0rem', fontSize: '1rem'}} filter filterElement={rowEditorFilterNameHeader} showFilterMenu={false}
+							headerStyle={{ width: '3rem', position: 'sticky' }} bodyStyle={{ textAlign: 'center' }} frozen headerClassName='surface-0'
```

Comment:
This is okay since there was already a style prop in place. We'll probably refactor this in the future.

Reply:
hard coding font sizes is going to effect the person's ability to resize from the config menu

Reply:
@oblodgett We tested this locally and the resizing from the config menu appears to still work as it did previously.